sale_order_html_sale_conditions/models/sale_order.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    @api.model
+    def _default_sale_conditions(self):
+        return self.env.company.default_sale_conditions or ""
+
+    sale_conditions = fields.Html(
```

Comment:
We may want a translation option there

Reply:
Hum, I think it's good like this, the behavior is the same than the core field `note`.
I want to have the same behavior to be consistent.

Reply:
fine with me ;)